src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+:root {
+  --grey-color: #3d4e61;
+}
+
+.search-form {
+  display: flex;
+  width: 100%;
+  margin-top: 20px;
+  font-family: "Avenir", sans-serif;
+  color: var(--grey-color);
+}
+
+.search-label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.search-label:before {
+  content: "";
+  position: absolute;
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  z-index: 1;
+}
+
+.search-form__input {
+  display: flex;
+  width: 100%;
+  outline: none;
+  height: 70px;
+  font-family: inherit;
+  padding-left: 62px;
+  color: var(--grey-color);
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  border-radius: 4px;
+}
+
+.search-form__input__small {
+  width: 100%;
+  height: 42px;
+  outline: none;
+  padding-left: 33px;
+  font-family: inherit;
+  font-size: 14px;
+  color: var(--grey-color);
+  border: 1px solid #e1e7ed;
+  background: #fff;
```

Comment:
Consider using variables for repeating colors

Reply:
not fixed